src/libs/e2e/getIsE2e.web.ts

```diff
@@ -0,0 +1,10 @@
+export async function getIsE2e() {
+  // Read more: https://developer.mozilla.org/en-US/docs/Web/API/Navigator/webdriver
+  if (globalThis.navigator.webdriver) {
+    return globalThis.navigator.webdriver
```

Comment:
```suggestion
    return !!globalThis.navigator.webdriver
```
plutôt, comme en-dessous, non ?

Reply:
le if assure a typescript un true.